docs/services/vision/segmentation.md

```diff
@@ -218,6 +219,83 @@ The following parameters are available for a `detector_3d_segmenter`.
 
 Click **Save config** and proceed to [test your segmenter](#test-your-segmenter).
 
+## Configure an `obstacles_depth` segmenter
+
+This segmenter model is for depth cameras, and is best for motion planning with transient obstacles.
+If you have [configured the frame system](/services/frame-system/#configuration) for the components of your robot, then you can implement this model to identify well separated objects above a flat plane.
+
+To configure an `obstacles_depth` segmenter, first decide if you want to use the segmenter with or without intrinsic parameters.
+
+This is determined by the value of the configuration attribute `return_pcds`:
```

Comment:
Not true.. this is actually determined by **with_geometries**. 

return_pcds is a separate optional parameter (default false) that determines whether pointclouds should be included within the GeometryInFrame object. 

So if w_g and r_p were both true, you would get 10 GeometryInFrame objects, each with a geometry and a pointcloud inside. If w_g was false but r_p was true, you'd get a single GeometryInFrame object that would have a 1-point-pointcloud and a geometry (point).  When r_p is false, you get the same things just without the pointcloud. Geometry only.

Reply:
Ok tried to update!! Sorry for the confusion